src/couldBeSharedComponents/LayoutPanels/LayoutPanels.test.tsx

```diff
@@ -36,6 +36,7 @@ describe("Layout Panels default", async () => {
     await waitFor(() => {
       expect(screen.queryByText("right panel content")).not.toBeInTheDocument();
     });
+    expect(screen.getByText("main content")).toBeInTheDocument();
```

Comment:
Its worth in the default test to ensure the main content is showing during all the hiding and showing of panels. I forgot to add that before.